app/backend/src/couchers/servicers/references.py

```diff
@@ -58,19 +58,24 @@ def ListReferences(self, request, context):
 
             to_users = aliased(User)
             from_users = aliased(User)
+            # relevant_blocks = all_blocked_or_blocking_users(context.user_id)
```

Comment:
Wasn't sure if you should be able to see reviews from blocked/blocking users so you can know what they're saying about you (and others)